docs/modules/ROOT/nav.adoc

```diff
@@ -15,8 +15,9 @@
 
 .Guides
 * xref:advanced-scripting.adoc[Advanced Scripting]
-* xref:using-compact-serializer-generator.adoc[Using Compact Serializer Generator]
+* xref:using-compact-serializer-generator.adoc[Compact Serializer Generator]
 * xref:project-creator.adoc[Project Creator]
+* xref:config-safe.adoc[Secure Cluster Configuration]
```

Comment:
Would it make sense to revise it?
"Secure Cluster Configuration" sounds like the guide is for securing the cluster itself.